connectors/logger.py

```diff
@@ -32,41 +35,38 @@ class ColorFormatter(logging.Formatter):
 
     def __init__(self, prefix):
         self.custom_format = "[" + prefix + "][%(asctime)s][%(levelname)s] %(message)s"
-        super().__init__()
-
-    @cached_property
-    def debug_formatter(self):
-        return logging.Formatter(
-            fmt=self.GREY + self.custom_format + self.RESET, datefmt=self.DATE_FMT
-        )
-
-    @cached_property
-    def info_formatter(self):
-        return logging.Formatter(
-            fmt=self.GREEN + self.custom_format + self.RESET, datefmt=self.DATE_FMT
-        )
-
-    @cached_property
-    def warning_formatter(self):
-        return logging.Formatter(
-            fmt=self.YELLOW + self.custom_format + self.RESET, datefmt=self.DATE_FMT
-        )
-
-    @cached_property
-    def error_formatter(self):
-        return logging.Formatter(
-            fmt=self.RED + self.custom_format + self.RESET, datefmt=self.DATE_FMT
-        )
-
-    @cached_property
-    def critical_formatter(self):
-        return logging.Formatter(
-            fmt=self.BOLD_RED + self.custom_format + self.RESET, datefmt=self.DATE_FMT
-        )
+        super().__init__(datefmt=self.DATE_FMT)
+
+    def converter(self, timestamp):
+        tzinfo = tzlocal.get_localzone()
+        dt = datetime.datetime.fromtimestamp(timestamp, tzinfo)
+        return dt.astimezone(pytz.UTC)
```

Comment:
I think we can avoid adding `pytz` to the mix, `datetime` lib supports all we need, see example:
https://github.com/elastic/connectors/blob/644c4e018043e24e529b8dadce7665e7da47f5a1/connectors/utils.py#L97-L102